File: ocaml/dist.ml
let dpow a b = (a -. b) ** 2.0;;

let distanciadp xp1 yp1 xp2 yp2 = sqrt ((dpow xp2 xp1) +. (dpow yp2 yp1));;
    

let (xp1, yp1) = Scanf.scanf "%f  %f\n" (fun a b ->(a, b));;
let (xp2, yp2) = Scanf.scanf "%f  %f\n" (fun c d ->(c, d));;

let res = distanciadp xp1 yp1 xp2 yp2;;
Printf.printf "%.4f\n" res;;
  

File: kotlin/dist.kt
import java.util.Scanner

fun distanciadp( xp1: Double,  yp1: Double,  xp2: Double,  yp2: Double ): Double {
    var base : Double;
    base = Math.pow((xp2 - xp1) ,2.0) + Math.pow((yp2 - yp1), 2.0);
    return ( Math.sqrt( base ));
}

fun main(args: Array<String>){
    var (xp1, yp1) = readLine()!!.split(' ').map(String::toDouble )
    var (xp2, yp2) = readLine()!!.split(' ').map(String::toDouble )
    var res = distanciadp(xp1, yp1, xp2, yp2)
    println("%.4f".format(res))
}